basket/src/components/Profile.tsx

```diff
@@ -0,0 +1,28 @@
+type Session = {
+  loginUser: {
+    id: number;
+    name: string;
+  };
+  cart: Array<{ id: number; name: string; price: number }>;
+};
+
+type MyProps = {
+  session: Session;
+  logout: () => void;
+};
+
+export const Profile = ({ session, logout }: MyProps) => {
+  return (
+    <div>
+      <p>Logged in as: {session.loginUser.name} ({session.loginUser.id})</p>
+      <ul>
+        {session.cart.map((ele) => (
+          <li style={{listStyle: 'none'}} key={ele.id}>
+            <strong>{ele.name}</strong>
+            <span>{ele.price}</span>
+          </li>))}
+      </ul>
```

Comment:
컴포넌트를 조금 추상화해주면 좋을 것 같은 부분!
아래와 같이 CartList 컴포넌트를 만들어서 사용해보기!

```suggestion
    <CartList items={session.cart} />
```